.travis.yml

```diff
@@ -42,3 +42,13 @@ deploy:
   on: &2
     branch: master
     condition: $DEPLOY=true
+- provider: codedeploy
+  access_key_id: AKIAINSAHPOLGXFFYGMQ
```

Comment:
can we copy this from access_key_id?
https://github.com/openregister/openregister-java/blob/80730666bd339bd19621d36b5f5a35e1e34d2ad8/.travis.yml#L35